e2e/resources/workbench-config.ts

```diff
@@ -43,7 +43,7 @@ const staging = {
   userEmailDomain: '@staging.fake-research-aou.org',
   collaboratorUsername: 'puppetcitester4@staging.fake-research-aou.org',
   defaultCdrVersionName: 'Synthetic Dataset v3',
-  altCdrVersionName: 'Synthetic Dataset v2',
+  altCdrVersionName: 'Synthetic Dataset v3',
```

Comment:
Would it be possible to add an alt version to Stable and Staging?  If we ever run e2e tests there, they would fail due to the lack of an alternate.

Reply:
We can, but it will be a duplicate of the other.

Reply:
That would be fine for test purposes.

Reply:
Done.